developers/subgraph.md

```diff
@@ -399,7 +399,7 @@ First we need to create our ID using the parameters define in the [Querying time
 ```GraphQL
 {
   dailyBorrow(id: "1644451200") {
-  	count
+   count
```

Comment:
I'm not sure if this is what we want, since this will affect how the code is displayed in Docs

I think we should keep tabs here 🤔